src/configure.rs

```diff
@@ -75,6 +75,10 @@ impl Context {
             }))
             .await;
 
+        if let Err(err) = res.as_ref() {
+            progress!(self, 0, Some(err.to_string()));
+        }
```

Comment:
imu, this may result in "progress = 0 = error" to be emitted twice - once here and one time at the end of `inner_configure()` - maybe we should remove the corresponding `progress!(self, 0, ...)` call from `inner_configure` somehow.